search-commons/src/main/java/no/unit/nva/indexingclient/models/IndexDocument.java

```diff
@@ -71,6 +71,7 @@ public String getDocumentIdentifier() {
     public IndexRequest toIndexRequest() {
         return new IndexRequest(getIndexName())
             .source(serializeResource(), XContentType.JSON)
+            .routing("0") // Target shard ID
```

Comment:
Perhaps we should document this and/or create a follow-up ticket to improve it? Essentially hard-coding the number of shards, buried deep inside a model.

Reply:
An inconvenient truth, hidden deep down the stack.
But that is ok, as you are working for the state now. 🤷‍♂️

Reply:
But thank god, this is not politics, we'll fix it.